Python/README.md

```diff
@@ -1,4 +1,8 @@
-#Python
+##Python Samples
+
+[![Build Status](https://travis-ci.org/alfpark/azure-batch-samples.svg?branch=master)](https://travis-ci.org/alfpark/azure-batch-samples)
```

Comment:
Also the coveralls button is broken. It doesn't like that it's in a README that's not in the root of the repo.